### DISCUSSION TITLE
Elastic Net like problem

### DISCUSSION THREAD

**Discussion by raypenper** [osqp/osqp#654] [Mathematical Modeling]
Can QSQP solve the Elastic Net like problem? It has the form $$min_{x}||Ax - b||^2_2+\lambda||x||_1+\mu||x^2-y||^2_2$$ where y is a given vector. This problem is basically a lasso problem with L2 regularization, but it has a non-linear term $x^2$ and a reference solution $y$. Anyone know how to solve the problem? Thanks a lot.

**Comment by imciner2** [comment]
Hi @raypenper, just so I understand better. $y$ is a constant that is given before solving the problem and not an optimization variable? Making this regularization term trying to minimize the difference between $x^2$ and $y$?

**Reply by raypenper** [reply]
Yes, that's correct. I think the last term is not convex and maybe impossible to solve. But the problem has practical application.
